packages/es-components/src/components/containers/modal/Modal.specs.js

```diff
@@ -67,14 +67,18 @@ describe('when ESC is pressed', () => {
     const { container } = renderModal({ onEnter: jest.fn(), onHide });
 
     fireEvent.keyDown(container, { keyCode: 27 });
-    expect(onHide).toHaveBeenCalled();
+    wait(() => {
+      expect(onHide).toHaveBeenCalled();
+    });
   });
 
   it('does not invoke onHide when escapeExits is false', () => {
     const onHide = jest.fn();
     const { container } = renderModal({ onHide, escapeExits: false });
 
     fireEvent.keyDown(container, { keyCode: 27 });
-    expect(onHide).not.toHaveBeenCalled();
+    wait(() => {
```

Comment:
Isn't `wait()` async?

Reply:
it is, but it doesn't matter since I am doing the check inside the callback.